dev/NavigationView/NavigationView.cpp

```diff
@@ -1483,15 +1485,35 @@ void NavigationView::UpdatePaneLayout()
         const auto totalAvailableHeight = [this]() {
             if (const auto &paneContentRow = m_itemsContainerRow.get())
             {
-                // 20px is the padding between the two item lists
-                if (const auto &paneFooter = m_leftNavFooterContentBorder.get())
+                const double itemsContainerMargin = [this]() {
+                    if (const auto itemsContainer = m_itemsContainer.get())
+                    {
+                        const auto margin = itemsContainer.Margin();
+                        return margin.Top + margin.Bottom;
+                    }
+                    return 0.0;
+                }();
+                auto availableHeight = paneContentRow.ActualHeight() - itemsContainerMargin;
+
+                // The -21 below is to account for the separator height that we need to subtract.
+                if (PaneFooter())
+                {
+                    availableHeight -= 21;
```

Comment:
>21 [](start = 39, length = 2)

Is there a way to get this value programmatically instead using a magic number?

Reply:
I'm afraid not really, I wasn't able to "find" this in the layout, (actual)height of the separator is 6, however it still takes up that place.

Reply:
At the very least we should use a named constant